convokit/model/corpus.py

```diff
@@ -905,54 +946,56 @@ def reinitialize_index(self):
         new_index.version = old_index.version
         self.meta_index = new_index
 
-    def merge(self, other_corpus, warnings: bool = True):
+    @staticmethod
+    def merge(primary: "Corpus", secondary: "Corpus", warnings: bool = True):
         """
-        Merges this corpus with another corpus.
+        Merges primary and secondary, creating a new Corpus with their combined data.
```

Comment:
In general, I would refer to the corpora as `primary corpora` and `secondary corpora` in the documentation for readability, rather than just refer to them by `primary` and `secondary`.